coordinates/sites.json

```diff
@@ -564,13 +564,16 @@
         "aliases": [
              "LSST",
              "LSST 8.4m",
-             "Rubin"
+             "Rubin",
+             "rubin",
+             "Simonyi Survey Telescope",
+             "Rubin:Simonyi"
         ]
     },
-    "rubin_aux": {
+    "rubin_auxtel": {
```

Comment:
Would it be possible to not change the primary key? It's a list of observatories or telescopes, so the fact that it's a "tel" is implied, I think. We can have as many aliases as make sense, but we shouldn't change the primary key unless it's absolutely necessary.

Reply:
Ok. Reverted. I changed it because no-one at Rubin calls it this and everyone calls it `auxtel`.

Reply:
Presumably you want me to revert the `rubin` -> `rubin_simonyi` key change as well then?

Reply:
Yes, I think that would be nice. People think of the "Rubin" as the main telescope of the observatory. That's probably not correct, but so common, that it might be OK to keep that as key. People using this from astropy as intended won't know the difference (if it's an alias they are searching for or the main key), but it's a public json file and we don't know who else uses it for what purpose, so better to be careful.

Thanks!

Reply:
Done.